app/src/main/java/com/schibsted/tapdadev/MainPresenter.java

```diff
@@ -68,6 +72,11 @@ private void show(int developer) {
     private void show(View view) {
```

Comment:
Completely agree, I changed the show method to rollDeveloper that means show and hide one developer.